src/main/java/org/apache/commons/csv/CSVParser.java

```diff
@@ -799,7 +814,9 @@ CSVRecord nextRecord() throws IOException {
 
     /**
      * Returns a sequential {@code Stream} with this collection as its source.
-     *
+     * <p>
+     * If the parser is closed, the stream will not produce any more values.
+     * See the comments in {@link iterator()}.
```

Comment:
Close HTML tags.

Reply:
Done.
I was basing it on https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format which doesn't close the tags (I assume javadocs inherited the 'tag omission' behavior from  html).

Reply:
> Done. I was basing it on https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format which doesn't close the tags (I assume javadocs inherited the 'tag omission' behavior from html).

In general, you can look at the rest of the file to get an idea of what style to follow.